### PR TITLE
Create an effect for Cinder Gaze

### PR DESCRIPTION
**Pull request by LebombJames**

### CODE REVIEW

File: packs/spells/cinder-gaze.json [hunk 1]
```diff
@@ -20,17 +20,34 @@
...
+                "hC5GKInrfS2NyRMF": {
+                    "applyMod": true,
+                    "type": {
+                        "categories": [],
+                        "subtype": "",
+                        "value": ""
+                    },
+                    "value": "1"
+                }
+            }
```
Comment (TikaelSol) [inline packs/spells/cinder-gaze.json:33]: This doesn't deal damage

File: packs/spell-effects/effect-cinder-gaze.json [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+                "predicate": [
+                    "action:recall-knowledge",
+                    "self:effect:cinder-gaze:3"
+                ],
+                "removeAfterRoll": true,
+                "selector": "skill-check"
+            }
+        ],
+        "source": {
+            "value": ""
```
Comment (TikaelSol) [inline packs/spell-effects/effect-cinder-gaze.json:63]:
```suggestion
            "value": "Pathfinder Dark Archive"
```

File: packs/spells/cinder-gaze.json (hunk 1 above)
Comment (TikaelSol) [reply packs/spells/cinder-gaze.json:33]: A TempHP rule element.

Comment (TikaelSol) [reply packs/spells/cinder-gaze.json:33]: The issue with setting damage on the spell is bonuses to damage that apply to spells, like inspire courage, would affect this too.

Comment (LebombJames) [reply packs/spells/cinder-gaze.json:33]: That makes sense, but I don't know of a way to do "1 + spellcasting ability modifier" in a formula. Does an `@attribute` exist for the value of your spellcasting ability mod?

Comment (LebombJames) [reply packs/spells/cinder-gaze.json]: Well, it works, if only moderately hacky

**Pull request merged**